tasks.py

```diff
@@ -824,6 +824,26 @@ def webpack(clean=False, watch=False, dev=False):
 
 
 @task()
+def webpack_admin(clean=False, watch=False):
+    """Build static assets with webpack."""
+    build_js_config_files()
+    if clean:
+        clean_assets()
```

Comment:
I think clean is the parameter that you pass as True if you want to invoke clean_assets. It's not saying if it is already clean clean again. This is how the webpack method for the osf is structured but I can change it if it doesn't make sense. What do you think?